services/ui-src/src/components/reports/ModalOverlayReportPage.tsx

```diff
@@ -41,20 +48,54 @@ export const ModalOverlayReportPage = ({ route }: Props) => {
     addEditEntityModalOnCloseHandler();
   };
 
+  // delete modal disclosure and methods
+  const {
+    isOpen: deleteEntityModalIsOpen,
+    onOpen: deleteEntityModalOnOpenHandler,
+    onClose: deleteEntityModalOnCloseHandler,
+  } = useDisclosure();
+
+  const openDeleteEntityModal = (entity: EntityShape) => {
+    setSelectedEntity(entity);
+    deleteEntityModalOnOpenHandler();
+  };
+
+  const closeDeleteEntityModal = () => {
+    setSelectedEntity(undefined);
+    deleteEntityModalOnCloseHandler();
+  };
+
   return (
     <Box data-testid="modal-overlay-report-page">
       {verbiage.intro && (
         <ReportPageIntro
           text={verbiage.intro}
-          accordion={accordionVerbiage.formIntro}
+          accordion={accordionVerbiage.MLR.formIntro}
         />
       )}
       <Box sx={sx.dashboardBox}>
         <Heading as="h3" sx={sx.dashboardTitle}>
           {dashTitle}
         </Heading>
-        {reportFieldDataEntities.length === 0 && (
+        {reportFieldDataEntities.length === 0 ? (
           <Box>{verbiage.emptyDashboardText}</Box>
+        ) : (
+          <>
```

Comment:
I think we can get rid of this tag here

Reply:
agreed, do not need the wrapper `<></>` tag